packages/@aws-cdk/aws-eks/lib/spot-interrupt-handler.ts

```diff
@@ -3,173 +3,13 @@ export enum LifecycleLabel {
   SPOT = 'Ec2Spot',
 }
 
-const DEFAULT_NODE_SELECTOR = { lifecycle: LifecycleLabel.SPOT };
-
-export function spotInterruptHandler(nodeSelector: { [name: string]: string } = DEFAULT_NODE_SELECTOR) {
-  return [
-    {
-      kind: "ClusterRole",
-      apiVersion: "rbac.authorization.k8s.io/v1",
-      metadata: {
-        name: "node-termination-handler",
-        namespace: "default"
-      },
-      rules: [
-        {
-          apiGroups: [
-            "apps"
-          ],
-          resources: [
-            "daemonsets"
-          ],
-          verbs: [
-            "get",
-            "delete"
-          ]
-        },
-        {
-          apiGroups: [
-            ""
-          ],
-          resources: [
-            "*"
-          ],
-          verbs: [
-            "*"
-          ]
-        },
-        {
-          apiGroups: [
-            "rbac.authorization.k8s.io"
-          ],
-          resources: [
-            "*"
-          ],
-          verbs: [
-            "*"
-          ]
-        },
-        {
-          apiGroups: [
-            "apiextensions.k8s.io"
-          ],
-          resources: [
-            "customresourcedefinitions"
-          ],
-          verbs: [
-            "get",
-            "list",
-            "watch",
-            "create",
-            "delete"
-          ]
-        }
-      ]
-    },
-    {
-      apiVersion: "v1",
-      kind: "ServiceAccount",
-      metadata: {
-        name: "node-termination-handler"
-      }
-    },
-    {
-      kind: "ClusterRoleBinding",
-      apiVersion: "rbac.authorization.k8s.io/v1",
-      metadata: {
-        name: "node-termination-handler",
-        namespace: "default"
-      },
-      subjects: [
-        {
-          kind: "ServiceAccount",
-          name: "node-termination-handler",
-          namespace: "default"
+export function spotInterruptHandler() {
+  return {
+        chart: 'aws-node-termination-handler',
+        repository: 'https://aws.github.io/eks-charts',
```

Comment:
Does it make sense to specify a version here so this doesn't break unexpectedly?

Reply:
You are right. Will pin the current chart version to [1.7.3](https://github.com/aws/eks-charts/blob/e421230a2ad39c53e287ef6222d4438932460ebd/stable/aws-node-termination-handler/Chart.yaml#L4-L5).